Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Definitions/Options.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.Excel.ConvertToJSON.Definitions;
```

Comment:
Same here.

Reply:
In C# best practices namespaces go according to root namespace + folder structure.